STX.REST.RESTFulSense.Clients/Models/Brokers/StatusDetails/StatusDetail.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+namespace STX.REST.RESTFulSense.Clients.Models.Brokers.StatusDetails
+{
+    internal class StatusDetail
```

Comment:
Why this Model is under StatusDetails? I think it should be Models.Brokers.Errors